Makefile.common

```diff
@@ -91,18 +91,24 @@ prebuild-checks:
 #help Use MOCK_OPTS environment varible to pass down arbitrary mock options.
 build: prebuild-checks configemail $(SPECFILE) upstream $(SRPMFILE)
 	$(MOCK) $(SRPMFILE) --result=results/ --no-cleanup-after --uniqueext=$(PKG_NAME)
-	@perl $(TOPLVL)/projects/common/logcheck.pl results/build.log
-	@perl $(TOPLVL)/projects/common/count.pl results/build.log &> testresults
+	mkdir -p results/logs results/debuginfo
+	mv results/*.log results/logs
+	mv results/*-debuginfo*.rpm results/debuginfo/
+	@perl $(TOPLVL)/projects/common/logcheck.pl results/logs/build.log
+	@perl $(TOPLVL)/projects/common/count.pl results/logs/build.log &> testresults
 	@$(MAKE) checkblacklist
 
 #help build-nocheck: Same as 'make build', but do not run the package's test suite.
 build-nocheck: configemail $(SPECFILE) upstream $(SRPMFILE)
 	$(MOCK) $(SRPMFILE) --nocheck --result=results/ --no-cleanup-after --uniqueext=$(PKG_NAME)
-	@perl $(TOPLVL)/projects/common/logcheck.pl results/build.log
-	@perl $(TOPLVL)/projects/common/count.pl results/build.log &> testresults
+	mkdir -p results/logs
```

Comment:
This should be `mkdir -p results/logs results/debuginfo`, like you have for the `build` target, or else the mv command below for moving the debuginfo rpm will fail.

Reply:
woo ty!